scripts/west_commands/runners/core.py

```diff
@@ -649,6 +660,14 @@ def dev_id_help(cls) -> str:
                   target when multiple ones are available or
                   connected.'''
 
+    @classmethod
+    def extload_help(cls) -> str:
+        ''' Get the ArgParse help text for the --external loader option.'''
+        return '''Option to pass on to the underlying tool used
+                  by this runner. This must be given one time;
+                  the resulting arguments will be given to the tool
+                  in the order they appear on the command line.'''
```

Comment:
`External loader to be used by stm32cubeprogrammer to program the targeted external memory.